doc/man1/openssl-ca.pod

```diff
@@ -54,14 +54,14 @@ B<openssl> B<ca>
 [B<-create_serial>]
 [B<-rand_serial>]
 [B<-multivalue-rdn>]
-[B<-rand> I<file>]
+[B<-rand> I<files>]
 [B<-writerand> I<file>]
 [B<-sm2-id> I<string>]
 [B<-sm2-hex-id> I<hex-string>]
 
 =head1 DESCRIPTION
 
-The B<ca> command is a minimal CA application. It can be used
+The B<openssl ca> command is a minimal CA application. It can be used
```

Comment:
Basically for all of them...

Reply:
Yes.  Try to remove all instances of ```B<openssl foo>``` inside the manpage foo.pod  Because it jars with the links ```L<openssl-bar>``` and because also there are several instances where the text just has ```B<foo>``` which I tried to point out.